docs/utilities/sqs_large_message_handling.md

```diff
@@ -180,4 +180,31 @@ processing.
             return "ok";
         }
     }
+    ```
+
+## Passing custom S3Client
+
+If you need to pass custom S3Client such as region to the SDK, you can pass your own `S3Client` to be used by utility either for
```

Comment:
Wording change.

"If you require customisations to the default S3Client, you can create your own S3Client and pass it to ......"